### ISSUE TITLE
Touch swipe gestures do not work on fixed layout books

### ISSUE THREAD

**Issue by marek-lach** [johnfactotum/foliate#1225]
**Is your feature request related to a problem? Please describe.**
On touch-screen devices, such as laptops and tablets, it is possible to scroll pages with a finger swipe in EPUBs, but not PDFs.

**Describe the solution you'd like**
Make PDFs opened in Foliate also scrollable with a finger-swipe, not just by tapping using the bottom navigational arrow >

**Additional context**
I do realize this may be a limitation of the underlying PDF parsing library. As a result, it may be harder to implement in Foliate.

**Edit:** On account of that, tapping > < navigational arrows to turn the pages seems significantly slower with PDF documents, than with EPUBs.

**Comment by johnfactotum** [comment]
No, it's not a PDS.js problem. PDF.js is only used for rendering the pages themselves. Pagination is handled the same way as fixed-layout EPUBs, and currently the fixed-layout renderer is lacking many features.
